functionalTest/test/specs/regional.js

```diff
@@ -0,0 +1,13 @@
+import dashboard from "../pageobjects/dashboardPage";
+
+describe("Regional Insights", () => {
+  it("should have correct title", () => {
+    dashboard.open();
+    browser.url("/regional");
+    let title = "Scottish COVID-19 Statistics";
+    expect(browser).toHaveTitle(title);
+    expect(dashboard.imgLogo).toBeDisplayed();
+    expect(dashboard.headingTitle).toHaveText("Scottish COVID-19 Statistics");
+    expect(dashboard.headlineBanner).toHaveTextContaining("Regional Restrictions to help fight the pandemic");
```

Comment:
Selected this text to check for as it mentions "regional restrictions" which is unlikely to ever show up in the overview banner.